script.js

```diff
@@ -1,4 +1,89 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  if (
+    input == "scissors" ||
+    input == "stone" ||
+    input == "paper" ||
+    input == "Scissors" ||
+    input == "Stone" ||
+    input == "Paper"
+  ) {
+    var myOutputValue = playGame(input);
+    return myOutputValue;
+  } else if (input == null) {
+    alert("Please enter a valid value");
+  } else {
+    alert("Please enter a valid value");
+  }
+};
+var getRandomInteger = function () {
+  // Generate a decimal from 0 through max + 1.
+  // This decimal will be inclusive of 0 and exclusive of max + 1.
+  var randomDecimal = Math.random() * (2 + 1);
+
+  // Remove the decimal with the floor operation.
+  // The resulting integer will be from 0 through max, inclusive of 0 and max.
+  var resultInteger = Math.floor(randomDecimal);
+
+  return resultInteger;
+};
+
+var randomIntegerToItem = function (input) {
+  if (input == 0) {
+    var randomItem = "Scissors";
+  } else if (input == 1) {
+    var randomItem = "Paper";
+  } else if (input == 2) {
+    var randomItem = "Stone";
+  }
+  return randomItem;
+};
+
+var playGame = function (itemChosen) {
+  // get randomItem
+  var randomItem = getRandomInteger();
+  console.log(randomIntegerToItem(randomItem));
+  var chosenItem = reverseItems(itemChosen);
+  if (randomItem == chosenItem) {
+    console.log(randomItem);
+    var myOutputValue =
+      "This is a classic draw. You chose " +
+      itemChosen +
+      " and the computer chose " +
+      randomIntegerToItem(randomItem);
+  } else if (
+    (randomItem == 2 && chosenItem == 1) ||
+    (randomItem == 1 && chosenItem == 0) ||
+    (randomItem == 0 && chosenItem == 2)
```

Comment:
You might consider wanting to combine the getRandomInteger function with the randomIntegerToItem function to generate a number internally and outputting a hand from there. That way, you don't have to use numbers for the if statements and can just use strings (Which would be easier to understand at a glance)